File: android/src/main/java/com/plugin/flutter/zsdk/Orientation.java
package com.plugin.flutter.zsdk;

/**
 * Created by luis901101 on 2020-01-07.
 */
public enum Orientation
{
    PORTRAIT,
    LANDSCAPE;

    public static Orientation getValueOfName(String name){
        if(name == null) return null;
        if(name.equals(Orientation.PORTRAIT.name())) return Orientation.PORTRAIT;
        if(name.equals(Orientation.LANDSCAPE.name())) return Orientation.LANDSCAPE;
        return null;
    }
}

File: ios/Classes/Orientation.swift
//
//  Orientation.swift
//  zsdk
//
//  Created by Luis on 1/9/20.
//

import UIKit

enum Orientation: String, CaseIterable {
    case PORTRAIT = "PORTRAIT"
    case LANDSCAPE = "LANDSCAPE"

    public static func getValueOfName(name: String) -> Orientation {
        if(name == PORTRAIT.rawValue) {return PORTRAIT}
        if(name == LANDSCAPE.rawValue) {return LANDSCAPE}
        return LANDSCAPE
    }
}
